components/resources/brave_components_strings.grd

```diff
@@ -797,6 +797,10 @@
         You're viewing a WebTorrent page
       </message>
 
+      <message name="IDS_TEST_STRING_FOR_PLACEHOLDERS" desc="Only for test purpose. Don't need to translate">
```

Comment:
@simonhong, you can use `translateable="false"` attribute in the future if the string doesn't need to be translated.

Reply:
Got it, thanks!